Userland/Applications/TodoList/TaskListModel.cpp

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2021, Arthur Brainville <ybalrid@ybalrid.info>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "TaskListModel.h"
+
+#include <AK/JsonArray.h>
+#include <AK/JsonObject.h>
+#include <AK/JsonParser.h>
+#include <AK/JsonValue.h>
+
+#include <cstdio>
+
+void TaskListModel::load_from_disk(AK::String path)
+{
+    //TODO There's probably a better API in Serenity than the stdio for that
+    FILE* todofile = fopen(path.characters(), "r");
+    StringBuilder jsonBuilder;
+    if (todofile) {
+        for (;;) {
+            char buffer[1024];
+            if (!fgets(buffer, sizeof(buffer), todofile))
+                break;
+            jsonBuilder.append(buffer);
+        }
```

Comment:
There's `LibCore` which has the `Core::File` class. `File` objects have a `read_all` method. See `Userland/Applications/KeyboardSettings/main.cpp`, lines 71-76 for an example.

Reply:
Thanks for pointing it!

(The tests for LibJson are using the pattern above - should it also use LibCore::File instead of good-old stdio C library? 🙂)